DigitalLearningSolutions.Web/Views/MyAccount/MyDetailsCustomField.cshtml

```diff
@@ -1,10 +1,11 @@
-﻿@model DigitalLearningSolutions.Web.ViewModels.MyAccount.CustomFieldViewModel
-
-<div class="nhsuk-summary-list__row details-list-with-button__row">
-  <dt class="nhsuk-summary-list__key">
-    @Model.CustomPrompt@(Model.Mandatory ? ":" : " (optional):")
-  </dt>
-  <dd class="nhsuk-summary-list__value">
-    @Model.Answer
-  </dd>
-</div>
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model CustomFieldViewModel
+
+<div class="nhsuk-summary-list__row details-list-with-button__row">
+  <dt class="nhsuk-summary-list__key">
+    @Model.CustomPrompt@(Model.Mandatory ? ":" : " (optional):")
+  </dt>
+  <dd class="nhsuk-summary-list__value">
+    @Model.Answer
+  </dd>
+</div>
```

Comment:
Looks like this is changed to unix line endings - have you got the correct configuration for line endings?